docs/New-ExternalHelpCab.md

```diff
@@ -89,12 +92,18 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## INPUTS
 
 ### None
-This cmdlet does not take in input over the pipeline.
+You cannot pipe values to this cmdlet.
 ## OUTPUTS
 
 ### None
-This cmdlet does not output to the console. The only output is in the output folder specificed by the -OutputPath parameter.
+This cmdlet does not generate output.
+The cmldet saves its results in the output folder that the *OutputPath* parameter specifies.
 ## NOTES
 
 ## RELATED LINKS
 
+[Online Version:](https://github.com/PowerShell/platyPS/blob/master/docs/New-ExternalHelpCab.md)
+
+[New-ExternalHelp](New-ExternalHelp.MD)
+
+[New-MarkdownHelp](New-MarkdownHelp.md)
```

Comment:
These links in related links what makes the build unhappy. As I said, currently only `http*` or empty links would be accepted as valid. It's tracked by #140